#!/usr/bin/env python3


class ImageProcData:

    def __init__(self):
        self.serialID = 0

        self.xCoordMainBeaconCenter = 0
        self.yCoordMainBeaconCenter = 0
        self. xCoordMainBeaconLB = 0
        self.yCoordMainBeaconLB = 0
        self.xCoordMainBeaconUB = 0
        self.yCoordMainBeaconUB = 0

        self.xCoordSternBeaconCenter = 0
        self.yCoordSternBeaconCenter = 0
        self.xCoordSternBeaconLB = 0
        self.yCoordSternBeaconLB = 0
        self.xCoordSternBeaconUB = 0
        self.yCoordSternBeaconUB = 0

        self.xCoordBowBeaconCenter = 0
        self.yCoordBowBeaconCenter = 0
        self.xCoordBowBeaconLB = 0
        self.yCoordBowBeaconLB = 0
        self.xCoordBowBeaconUB = 0
        self.yCoordBowBeaconUB = 0

        self.xCoordFrontBeaconCenter = 0
        self.yCoordFrontBeaconCenter = 0
        self.xCoordFrontBeaconLB = 0
        self.yCoordFrontBeaconLB = 0
        self.xCoordFrontBeaconUB = 0
        self.yCoordFrontBeaconUB = 0
        return
